### ISSUE TITLE
Adding a custom visual queue for hit condition breakpoints 

### ISSUE THREAD

**Issue by cmoscofian** [mfussenegger/nvim-dap#1359]
### Problem Statement

Currently the `hit_condition` breakpoints, when available, have the same visualization as a regular breakpoint.
I think would be beneficial to add the ability to set it's own visual queue (sign and hl-group), similar to what has been done for `log_point`.
I might be wrong here, and if that is the case I apologize, but sounds simple enough to implement and has a fair bit of value.

### Possible Solutions

I think it would be useful in the context of loop debugs where mixed breakpoints are created, meaning some parts you only care on hit_conditions and others you might want to inspect every-time. It would be good being able to visualize that the breakpoints are correctly setup on a glance, not having to run and finding out the wrong one has been setup.

**Comment by mfussenegger** [comment]
One problem here is that the options can all be combined, you can have:

- simple breakpoints
- breakpoint with condition
- breakpoint with condition and hit condition
- breakpoint with log message
- breakpoint with log message and condition
- breakpoint with log message and hit condition
- breakpoint with log message, condition and hit condition

I'm not to keen on adding a separate hl-group for each combination.

**Comment by miroshQa** [comment]
Yeah, using vim.diagnostic.set for breakpoints would be a blast. You basically also get 'jump to next breakpoint' / 'jump to prev breakpoint' for free. Also, you can display a bunch of other info about breakpoints via virtual lines with zero effort
